### ISSUE TITLE
when xodr lanes have offset ,convertion to xml makes mistake

### ISSUE THREAD

**Issue by stanhua123** [daohu527/imap#58]
Hi, I have this problem: 
<lanes>
            <laneOffset s="0.0000000000000000e+0" **a="3.5000000000000000e+0**" b="0.0000000000000000e+0" c="0.0000000000000000e+0" d="0.0000000000000000e+0"/>

This lane have offset "a = 3.5"，when convert to xml and show in apollo ,the junction including the lane cannot show correctly
![image](https://user-images.githubusercontent.com/102218746/196368457-8ccf240a-cba6-4ad6-af86-fe37b55c299c.png)
I checked code and found in lanes.py ,left direction is -1 and right direction is 1,maybe this causes issues.
Thanks for helping !

**Comment by daohu527** [comment]
You use a xodr map to apollo map? left direction is used to identify the direction, and I have tested it on other maps, I suggest you first check if the xodr map is correct.

**Comment by stanhua123** [comment]
I use roadrunner to create the map,and use online viewer to check, seems no problem. And this only happens in junction when reference line has offset

**Comment by daohu527** [comment]
could you pls send the map to me daohu527@gmail.com, the offset maybe the problem, which currently not supported

**Comment by daohu527** [comment]
the lane offset is already considered in. 

I'm not sure if left-handedness and right-handed systems matter, but it looks like there's no problem here

https://github.com/daohu527/imap/blob/4c8713f2e48cf323a01cb9adcff621e56cf8b386/imap/lib/opendrive/road.py#L186

**Comment by stanhua123** [comment]
> the lane offset is already considered in.
> 
> I'm not sure if left-handedness and right-handed systems matter, but it looks like there's no problem here
> 
> https://github.com/daohu527/imap/blob/4c8713f2e48cf323a01cb9adcff621e56cf8b386/imap/lib/opendrive/road.py#L186

I tried the update , still couldn't convert rightly

**Comment by hzl0220** [comment]
Hi there! Have you fixed this problem?

**Comment by daohu527** [comment]
I did not find the cause of this problem, at least the offset of the reference line has been added.

Therefore, map data is required to determine further reasons

**Comment by hzl0220** [comment]
> I did not find the cause of this problem, at least the offset of the reference line has been added.
> 
> Therefore, map data is required to determine further reasons

Thank you, I just sent you an email about this issue, looking for the response :)

**Comment by daohu527** [comment]
thanks, I will check and feedback soon

**Comment by daohu527** [comment]
@hzl0220 I try the latest main code, the problem was fixed, but I haven't updated the version yet, I will upgrade to version 0.1.7 today.

You can pull the latest code to test locally
```
cd imap
python3 setup.py sdist
pip3 install dist/imap_box-0.1.6.tar.gz
```

**Comment by hzl0220** [comment]
> @hzl0220 I try the latest main code, the problem was fixed, but I haven't updated the version yet, I will upgrade to version 0.1.7 today.
> 
> You can pull the latest code to test locally
> 
> ```
> cd imap
> python3 setup.py sdist
> pip3 install dist/imap_box-0.1.6.tar.gz
> ```

Thank you so much, really enjoyed this work!

**Comment by daohu527** [comment]
Glad your problem was solved, the version updated to 0.1.7!
